sympy/stats/joint_rv_types.py

```diff
@@ -199,6 +203,14 @@ def MultivariateNormal(name, mu, sigma):
     exp(-(y - 4)**2/4)/(2*sqrt(pi))
     >>> marginal_distribution(X, X[0])(y)
     exp(-(y - 3)**2/4)/(2*sqrt(pi))
+    >>> # symbolic parameters
```

Comment:
Rather than having this as a comment, break the docstring section and just have some plain text that explains the example below.

Reply:
Sorry - how do you do that?

Reply:
Is it something like:
```
Examples
============

>>> ...

plain text

>>> ...
```

Reply:
> Is it something like:

Yes. When rendered in the html docs it show up as separate doctest blocks with some text in between.

Reply:
All done.